core/src/main/java/org/kiwix/kiwixmobile/core/main/MainMenu.kt

```diff
@@ -98,7 +100,14 @@ class MainMenu(
       else
         MenuItem.SHOW_AS_ACTION_NEVER
     )
-    tabSwitcher?.actionView?.setOnClickListener { menuClickListener.onTabMenuClicked() }
+    tabSwitcher?.actionView?.apply {
+      setOnClickListener { menuClickListener.onTabMenuClicked() }
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
We are using this code in multiple places, we should make an extension function for this and use that everywhere.

Reply:
Please elaborate more about the location, which class, and param.

Reply:
You should create the extension function in the `ViewExtension` class with the string param.

Reply:
setContentDescription is already a view function, so I am adding setToolTip